sites/frontend.preview.zooniverse.org.conf

```
@@ -0,0 +1,132 @@
+    rewrite ^/lab-best-practices/launch-rush$ https://help.zooniverse.org/best-practices/2-launch-rush permanent;
+    rewrite ^/lab-best-practices/the-long-haul$ https://help.zooniverse.org/best-practices/3-long-haul permanent;
+    rewrite ^/lab-best-practices/resources$ https://help.zooniverse.org/best-practices/4-resources permanent;
+
+    location /password/reset {
+        return 301 /reset-password;
+    }
+
+    location /projects/current {
+        return 301 /;
+    }
+
+    location /signup {
+        return 301 /accounts/register;
+    }
+
+    location /publications {
+        return 301 /about/publications;
+    }
+
+    location /api/events {
+        return 301 https://panoptes.zooniverse.org/api/events;
+    }
+
+    location /account/newsletters {
+        return 301 /settings/email;
+    }
+
+    location ~* /project(_|-)?builder$ {
+        return 301 /lab;
+    }
+
+    location /account/settings {
+        return 301 /settings;
+    }
+
+    location /home {
+        return 301 /;
+    }
+
+    location /team {
+        return 301 /about/team;
+    }
+
+    location ~* /project/.* {
+        return 301 /projects;
+    }
+
+    location /education {
+        return 301 /about/education;
+    }
+
+    location /contact {
+        return 301 /about/contact;
+    }
+
+    location ~* ^/projects/meredithspalmer/(cedar-creek-eyes-on-the-wild/?)(.*?)\/?$ {
+        return 301 /projects/forestis/$1$2$is_args$query_string;
+    }
+
+    location ~* ^/projects/karilint/the-cradle-of-mankind(/?)(.*?)\/?$ {
+        return 301 /projects/karilint/cradle-of-humanity$1$2$is_args$query_string;
+    }
+
+    location ~* ^/projects/cseidenstuecker/every-name-counts(/?)(.*?)\/?$ {
+        return 301 /projects/arolsen-archives/every-name-counts$1$2$is_args$query_string;
+    }
+
+    location ~* ^/projects/kevinesolberg/mapping-prejudice(/?)(.*?)\/?$ {
+        return 301 /projects/mappingprejudice/mapping-prejudice$1$2$is_args$query_string;
+    }
+
+    location ~* ^/projects/chiarasemenzin/maturity-of-baby-sounds(/?)(.*?)\/?$ {
+        return 301 /projects/laac-lscp/maturity-of-baby-sounds$1$2$is_args$query_string;
+    }
+
+    # ensure the js and CSS assets are served on the same or subdomain
+    location ~ \.(js|css)$ {
+        resolver 8.8.8.8;
+        proxy_pass  https://zooniversestatic.z13.web.core.windows.net/$proxy_path$request_uri;
+        include /etc/nginx/az-proxy-headers.conf;
+    }
+
+    # unsubscribe route uses redirects between panoptes and the UI code
+    # so needs it's own location block to handle the form submission POST
+    # and the GET page loading (PFE routing handles the path)
+    location /unsubscribe {
+        rewrite (?i)\.(jp(e)?g|gif|png|ico|txt|mp(3|4)|webm|og(a|g|m|v|x)|spx|opus|pdf|ttf|tar|gz|tgz|bz2|tbz2|zip)$ https://static.zooniverse.org/www.zooniverse.org$request_uri;
+
+        resolver 8.8.8.8;
```

use cloudflare dns not google
```suggestion
        resolver 1.1.1.1;
```